packages/fxa-auth-server/scripts/delete-account.js

```diff
@@ -20,6 +20,8 @@
 
 'use strict';
 
+const { StatsD } = require('hot-shots');
+const { Container } = require('typedi');
```

Comment:
Is it possible to add some tests for this in https://github.com/mozilla/fxa/tree/f4dc15a81d8bfa2df44b268367702cf7500c643f/packages/fxa-auth-server/test/scripts?

Reply:
Testing the delete-account script would require quite a bit of setup.  I'll file a jira for it; I don't want to block the fix because of it.